testbed/datasenders/syslog.go

```diff
@@ -67,9 +68,9 @@ func (f *SyslogWriter) Capabilities() consumer.Capabilities {
 
 func (f *SyslogWriter) Start() (err error) {
 	f.conn, err = net.Dial(f.GetEndpoint().Network(), f.GetEndpoint().String())
-	// udp not ack, can't use net.Dial to check udp server is ready, use sleep 1 second to wait udp server start
+	// udp not ack, can't use net.Dial to check udp server is ready, use sleep 2 second to wait udp server start
 	if f.network == "udp" {
-		time.Sleep(1 * time.Second)
+		time.Sleep(2 * time.Second)
 	}
```

Comment:
I would rather not do this, since it just masks the problem a bit more. We should aim for robustness. I suggest that we do not make this change now and instead do a proper fix which waits for the correct event: syslog udp receiver being ready to accept data. We can discuss how exactly we want to do that, there are some options:
- Use healthcheck extension and make sure it is responding with success before the tests are started
- Read Collector logs and wait for syslog receiver start message (a bit more fragile than previous approach)
- Maybe some other way.

Reply:
you are right. I plan to found a way to fix it.
the major problem (root cause test not stable) I think it's testbed used net.Dail to check server ready https://github.com/open-telemetry/opentelemetry-collector/blob/main/testbed/testbed/test_case.go#L198
it's not working for UDP server. it's make UDP server not ready send a lot of msg. and msg lost. make test failed

I think
1. opentelemetry-log-collection add some health check logic.
   (maybe like Redis. add ping/pong to check the server is working). will create an issue to discus
2. testbed/start-agent support datasenders customer health check



by the way. perf test sometimes fails, not means our syslog receiver logic has bug. the problem I think is udp server need to improve to make testbed known it's ok to start test.